tests/testthat/helper.R

```diff
@@ -26,3 +26,13 @@ expect_api_routes <- function(routes) {
     )
 }
 
+test_that <- function(desc, code) {
+    writeLines("")
+    writeLines(renv$header(paste(desc, "[begin]")))
+    call <- sys.call()
+    call[[1]] <- quote(testthat::test_that)
+    eval(call, envir = parent.frame())
+    writeLines(renv$header(paste(desc, "[ended]")))
+    writeLines("")
+    writeLines("")
+}
```

Comment:
Is this still needed? Or was it only for debugging?

Reply:
Just for debugging :-) I'll clean up the PR.